repos/system_upgrade/el8toel9/actors/checkarmbootloader/libraries/checkarmbootloader.py

```diff
@@ -0,0 +1,46 @@
+from leapp import reporting
+from leapp.libraries.common.config.architecture import ARCH_ARM64, matches_architecture
+from leapp.libraries.common.config.version import get_source_version, get_target_version
+from leapp.libraries.stdlib import api
+
+
+def _inhibit_upgrade():
+    title = 'Upgrade RHEL {} to RHEL {} not possible for ARM machines.'.format(
+        get_source_version(), get_target_version())
+    summary = ('Due to the incompatibility of the RHEL8 bootloader with a '
+               'newer version of kernel on RHEL9 a direct upgrade '
+               'cannot be performed for versions 9.5 and up.')
+
+    reporting.create_report([
+        reporting.Title(title),
+        reporting.Summary(summary),
+        reporting.ExternalLink(
+            title='Known issues for the RHEL 8.10 to RHEL 9.5 upgrade',
+            url='https://red.ht/upgrading-rhel8-to-rhel9-known-issues'),
+        reporting.Severity(reporting.Severity.HIGH),
+        reporting.Groups([reporting.Groups.INHIBITOR]),
+        reporting.Groups([reporting.Groups.SANITY]),
+        reporting.Remediation(hint=(
+            'To upgrade to a RHEL version 9.5 and up, first perform an upgrade '
+            'to version 9.4 using the leapp `--target` option and then perform '
+            'a minor dnf update after the upgrade')),
```

Comment:
```suggestion
            'To upgrade to the RHEL {} version, first in-place upgrade to RHEL 9.4 instead'
            ' using the leapp `--target=9.4` option. After you finish the upgrade - including'
            ' all required manual post-upgrade steps as well -'
            '  update to the newer minor version using the dnf tool. In case of using Red Hat'
            ' subscription-manager, do not forget to change the lock version to the newer one'
            ' or unset the version lock before using DNF to be able to perform the minor version update.'
            ' You can use e.g. `subscription-manager release --unset` command for that.'
            .format(get_target_version())
        )),
```

Reply:
Thanks a lot for expanding this! Fixed